.storybook/preview.tsx

```diff
@@ -8,12 +8,12 @@
 
 import { i18nInstance } from "@ndla/ui";
 import { Preview } from "@storybook/react";
-import React from "react";
 import { HelmetProvider } from "react-helmet-async";
 import { I18nextProvider } from "react-i18next";
 import { BrowserRouter } from "react-router-dom";
 import { DocsPage, DocsContainer } from "@storybook/addon-docs";
 import "./designmanual.scss";
+import "./styles.css";
```

Comment:
Lokalt bruker jeg kun denne. Ting ser med andre ord ikke helt likt ut i preview/prod, da de gamle global-stylingene fortsatt ligger der. Synes det er fint å ha det slikt inntil videre, men på sikt må det endres.

Reply:
Kunne det vært en idè å legge til en `//ToDo` her for å tydeliggjøre nettopp det? I tilfelle vi skulle få nytt team-medlem mellom når PR merges og når denne delen endres, tenker jeg på.

Reply:
Denne endres nok om dager og ikke uker, altså. Har den bare lokalt hos meg så lenge det kun er jeg som skal jobbe med det.